File: Labs/lab07/total_words.pl
#!/usr/bin/perl -w

use strict;

my $wordCount = 0;

while (my $line = <>) {
	$line =~ s/[^a-zA-Z]/ /g;
	my @words = split / /, $line;

	foreach my $word (@words) {
		if ($word ne "") {
			$wordCount += 1;
		}
	}
}

print "$wordCount words\n";

File: Labs/lab07/total_words.py
#!/usr/bin/python

import sys, re

wordCount = 0

for lines in sys.stdin:
	lines = re.sub("[^a-zA-Z]+", " ", lines)
	lines = re.sub("\'", " ", lines)
	words = lines.split(" ")
	
	for word in words:
		if word:
			#print (word)
			wordCount += 1

print ("%d words"%wordCount)